src/radix_connect/mobile/session/session.rs

```diff
@@ -7,19 +7,21 @@ pub struct Session {
     pub id: SessionID,
     pub origin: SessionOrigin,
     pub encryption_key: Exactly32Bytes,
-    // TBA: possibly will contain also the callback path
+    pub callback_path: String, // TBA: possibly will contain also the callback path
```

Comment:
hmm what is this? a URL? Why it this a `String`? We should be **very vary about `String`**, we should always try to use the most restrictive and desecriptive datatype for our data, which allows us developer to protect against bad values :)

Reply:
I like and try to follow Object Calisthenics by William Durand, it has been a source of inspiration for me for 15 years, great article and great "heuristics":

https://williamdurand.fr/2013/06/03/object-calisthenics/#3-wrap-all-primitives-and-strings

This is rule 3: *Wrap all primitives and strings*